pkg/rest/request/client.go

```diff
@@ -97,6 +98,12 @@ func (c Client) HTTPClient(client http.Client) Client {
 	return c
 }
 
+// Host sets the host for the client that is being built
```

Comment:
Does this only work when building the client? I think it sets the host for the client whenever it is used